### PR TITLE
Spark Rifle Coil and Laser

### PR DESCRIPTION
**Pull request by Danice123**
Also bugfix for elerium cost in vektor and autopistol
https://github.com/X2LongWarOfTheChosen/LWOTC_WeaponPack/issues/9
https://github.com/X2LongWarOfTheChosen/LWOTC_WeaponPack/issues/40

### CODE REVIEW

File: LWOTC_WeaponPack/Localization/Lasers/XComGame.int [hunk 1]
```diff
@@ -76,6 +76,13 @@ BriefSummary="Should their power fail them for some reason, the Templars rely on
 TacticalText="<Bullet/> The Autopistol gives the Templar the flexibility to combat the enemy in a conventional manner when not using their Psionic abilities.\n<Bullet/> Autopistols can be fired with a single action."
 AbilityDescName="autopistol"
 
+[SparkRifle_LS X2WeaponTemplate]
+FriendlyName="Laser Spark Rifle"
+FriendlyNamePlural="Laser Spark Rifles"
+BriefSummary=""
+TacticalText=""
```
Comment (TNychka) [inline LWOTC_WeaponPack/Localization/Lasers/XComGame.int]: Can you copy paste the conventional/mag/beam text? You have creative freedom here.

File: LWOTC_WeaponPack/Localization/Coils/XComGame.int [hunk 2]
```diff
@@ -88,6 +88,13 @@ BriefSummary="Should their power fail them for some reason, the Templars rely on
 TacticalText="<Bullet/> The Autopistol gives the Templar the flexibility to combat the enemy in a conventional manner when not using their Psionic abilities.\n<Bullet/> Autopistols can be fired with a single action."
 AbilityDescName="autopistol"
 
+[SparkRifle_CG X2WeaponTemplate]
+FriendlyName="Coil Spark Rifle"
+FriendlyNamePlural="Coil Spark Rifles"
+BriefSummary=""
+TacticalText=""
```
Comment (TNychka) [inline LWOTC_WeaponPack/Localization/Coils/XComGame.int]: Can you copy paste the conventional/mag/beam text? You have creative freedom here.

**Review by TNychka — CHANGES_REQUESTED** [review]

File: LWOTC_WeaponPack/Localization/Coils/XComGame.int (hunk 2 above)
Comment (Danice123) [reply LWOTC_WeaponPack/Localization/Coils/XComGame.int]: I just bought the spark DLC actually, I think that the text is missing from my localization files. I can't find it. @Anthogator was going through and changing names, and doing a good job of it too, I'd rather he just do that here as well.

Comment (Anthogator) [reply LWOTC_WeaponPack/Localization/Coils/XComGame.int]: Haha okay, I'll go in and make some new localization text for coil and laser spark rifles

Comment (Danice123) [reply LWOTC_WeaponPack/Localization/Coils/XComGame.int]: Honestly though, I swear I can't find the base game localization text for the guns. I've searched everything.

**Comment by Danice123** [comment]
@Anthogator Found the base game localization

**Review by TNychka — APPROVED** [review]

**Pull request merged**